src/indra_cogex/sources/chembl/__init__.py

```diff
@@ -91,7 +91,7 @@ def get_relations(self) -> Iterable[Relation]:
                 "has_indication",
                 {
                     "source": self.name,
-                    "max_phase:int": max_phase,
+                    "max_phase:float": max_phase,
```

Comment:
why float? to handle nans? typically this is 0-4

Reply:
Because some phases are =0.5

Reply:
See https://chembl.gitbook.io/chembl-interface-documentation/frequently-asked-questions/drug-and-compound-questions#what-is-max-phase